ArcSlider/ArcSlider.js

```diff
@@ -206,10 +223,11 @@ const ArcSliderBase = kind({
 		style: ({radius, style}) => {
 			const size = ri.scaleToRem(radius * 2);
 			return {...style, height: size, width: size};
-		}
+		},
+		circleRadius: ({skin, isFocused}) => skin === 'silicon' && isFocused ? 20 : 15
```

Comment:
As mentioned above, please make the handler expand for all other themes when focus. There is no GUI guide of Gallium and other for ArcSlider